lib/hanami/assets/bundler.rb

```diff
@@ -116,8 +128,11 @@ def generate_manifest
 
       # @since 0.1.0
       # @api private
-      def store_manifest(asset, target)
-        @manifest[_convert_to_url(::File.expand_path(asset))] = _convert_to_url(::File.expand_path(target))
+      def store_manifest(asset, target, sri_value)
+        @manifest[_convert_to_url(::File.expand_path(asset))] = {
+          target: _convert_to_url(::File.expand_path(target)),
```

Comment:
The DigestManifest is a map of assets to their respective digests:

```json
{"/assets/application.js":"/assets/application-123abc.js"}
```

I have extended this to store the SRI attribute value:

```json
{"/assets/application.js":{"target":"/assets/application-123abc.js","sri":"sha256-def456"}}
```

I think that it would be a good idea to drop the `target` for the actual MD5 checksum, generating the asset digest on the fly:

```json
{"/assets/application.js":{"md5":"123abc","sri":"sha256-def456"}}
```

This means that the DigestManifest has a consistent purpose of storing digest/hashes/checksums.

Thoughts?